quiz/management/commands/ensure_preview_key.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this line is redundant. See https://stackoverflow.com/q/14083111.

Reply:
Thanks! I didn't even notice it when I copy-pasted one of the other commands. If anyone wants to submit a PR removing it from other files, I'd be happy to merge it. Fixed it in this PR now and will merge it.